char-sheet/src/components/EquipmentTable.tsx

```diff
@@ -60,7 +61,25 @@ function EquipmentTable({equipment, container, service, editable=EditMode.Live}:
           items.map((item, i) => {
             return <tr className='hover tooltip tooltip-left w-full' data-tip={lookupDescription(item.name)} key={i}>
               <td className='w-full text-left'>
-                { item.name }
+                {item.custom ? (
+                  editingIndex === i ? (
+                    <input
+                      type='text'
+                      className='input input-bordered w-full'
```

Comment:
The input field is *super* chunky. Have you tested this? Try `input-sm` as className to at least be a smaller size. Maybe also try `py-0` (padding y to zero) on the parent `tr` if the current row is getting edited - it might get everything to line up a little better.